src/screens/Accessibility/index.tsx

```diff
@@ -162,6 +165,20 @@ export default function Accessibility() {
         </span>
 
         <ThemeSelector />
+        <span className="setting">
+          <label className={classNames('toggleWrapper', { isRTL: isRTL })}>
+            <ToggleSwitch
+              value={allTilesInColor}
+              handleChange={() => {
+                setAllTilesInColor(!allTilesInColor)
+              }}
+              title={t(
+                'accessibility.allTilesInColor',
```

Comment:
Please use camel_case here `accessibility.all_titles_in_color`.

Also, please run `yarn i18next` after changing that so it updates all the locales for all languages.